cmake/NVBenchExports.cmake

```diff
@@ -1,14 +1,28 @@
 macro(nvbench_generate_exports)
+  set(nvbench_build_export_code_block "")
+  set(nvbench_install_export_code_block "")
+
+  if (NVBench_ENABLE_NVML)
+    string(APPEND nvbench_build_export_code_block
+      "include(\"${NVBench_SOURCE_DIR}/cmake/NVBenchNVML.cmake\")\n"
```

Comment:
Using [=[ ]=] allows us to remove string escapes.

    string(APPEND nvbench_build_export_code_block
      [=[ include("${NVBench_SOURCE_DIR}/cmake/NVBenchNVML.cmake") 
      ]=])

Reply:
Good point, I always forget about those.

I think I like it as-is with the explicit newline, escapes, etc, I worry someone might be tempted to "clean this up" and accidentally remove the newline.